recipes/install.rb

```diff
@@ -10,60 +10,69 @@
 group = node[:bfd][:group]
 source_code_location = "#{Chef::Config[:file_cache_path]}/bfd"
 
-%w(ruby-dev ruby automake autoconf gcc g++ git make).each do |pkg|
+%w(checkinstall fakeroot automake autoconf gcc g++ git make).each do |pkg|
   package pkg do
     action :upgrade
   end
 end
 
-gem_package 'fpm' do
-  gem_binary '/usr/bin/gem'
-  action :install
-end
-
 git source_code_location do
    repository node[:bfd][:repo][:url]
    revision node[:bfd][:repo][:sha]
    action :sync
-   notifies :run, 'bash[buildtools_bfd]', :immediately
    not_if { ::File.exist?(target_filepath) }
 end
 
 bash 'buildtools_bfd' do
   cwd source_code_location
   code './autogen.sh'
-  action :nothing
-  notifies :run, 'bash[configure_bfd]', :immediately
+  action :run
+  not_if { ::File.exist?(target_filepath) }
 end
 
 bash 'configure_bfd' do
   cwd source_code_location
-  code "./configure --prefix=#{node[:bfd][:bin_dir]}/bfd-build"
-  action :nothing
-  notifies :run, 'bash[make_bfd]', :immediately
+  code './configure --prefix=/usr/local'
```

Comment:
This was previously using `#{node[:bfd][:bin_dir]}/bfd-build` (which was in the [chef cache dir](https://github.com/bloomberg/openbfdd-cookbook/blob/33c10ee5caf543b0b68e2f98dc3866af91078e78/attributes/default.rb#L17) probably incorrectly). Should we continue to use `node[:bfd][:bin_dir]` but point it to `/usr/local`?